asmcomp/linscan.ml

```diff
@@ -41,69 +34,73 @@ type class_intervals =
     mutable ci_fixed: IntervalSet.t;
     mutable ci_active: IntervalSet.t;
     mutable ci_inactive: IntervalSet.t;
-    mutable ci_spilled: SpilledSet.t;
-    mutable ci_free_slots: IntSet.t; (* stack slots available for reuse *)
+    mutable ci_spilled:
+      (* spilled stack slots (reg.loc = Stack (Local n)) still in use *)
+      IntervalSet.t;
```

Comment:
By using `IntervalSet` instaed of `SpilledSet` we lose the property that we always reuse the smallest possible stack slot, but perhaps we don't care.

Reply:
Is that true? I thought that this property was ensured by taking the min element of `SlotSet`, in a different part of the code.

Reply:
Argh, you are right!